src/App/Header/Connect.svelte

```diff
@@ -11,10 +12,15 @@
   import Floating from "@app/components/Floating.svelte";
   import Icon from "@app/components/Icon.svelte";
   import Link from "@app/components/Link.svelte";
+  import TextInput from "@app/components/TextInput.svelte";
 
+  $: customPort = httpd.api.port.toString();
```

Comment:
To make this sync between multiple open tabs, we'd have to get it from local storage in `lib/httpd.ts` instead of the api client.

Reply:
Handling this here https://github.com/radicle-dev/radicle-interface/issues/839